src/autocomplete/popup.js

```diff
@@ -114,11 +122,21 @@ var AcePopup = function(parentNode) {
         var row = popup.getRow();
         var t = popup.renderer.$textLayer;
         var selected = t.element.childNodes[row - t.config.firstRow];
-        if (selected !== t.selectedNode && t.selectedNode)
+        var el = document.activeElement; // Active element is textarea of main editor
+        if (selected !== t.selectedNode && t.selectedNode) {
             dom.removeCssClass(t.selectedNode, "ace_selected");
+            el.removeAttribute("aria-activedescendant");
+            t.selectedNode.id = "ace_line_" + t.selectedNode.id.split(":")[1];
```

Comment:
Why do we need this? Why can't we set `ace_line_{index}` once?

Reply:
Because of the line 135 in the same file ->
`selected.id = ariaId;` 
While moving through the popup list, the selected element gets assigned a special id which is used as the active descendant from the main editor textarea (which is the active element). If the line is unselected, we need to set its id back to its original value.

Reply:
I removed the second "ace_line_{index}" id. We now only have the id that I set for aria.